### ISSUE TITLE
Add digitalPinToInterrupt()

### ISSUE THREAD

**Issue by ShawnHymel** [sparkfun/SparkFun_APDS-9960_Sensor_Arduino_Library#1]
As per new Arduino library, digitalPinToInterrupt() needs to be added to attachInterrupt().

**Comment by BigSamosa** [comment]
Do I change this in the loop, or in the initialize isr? or both? Every example from this library works flawlessly except the gesture sensor example. Seems to work fine, but nothing is read.

**Comment by jcwren** [comment]
Yeah, 18 months later, and this still isn't fixed, and it bit me trying to run it on a Teensy LC. Line 73 should be changed to read

`attachInterrupt(digitalPinToInterrupt(APDS9660_INT), interruptRoutine, FALLING);`
